flathunter/abstract_crawler.py

```diff
@@ -193,6 +196,71 @@ def resolve_geetest(self, driver):
             driver.refresh()
             raise
 
+    @backoff.on_exception(wait_gen=backoff.constant,
+                        exception=CaptchaUnsolvableError,
+                        max_tries=3)
+    def resolve_awsawf(self, driver):
+        """Resolve AWS WAF Captcha"""
+
+        # Intercept background network traffic via log sniffing
+        sleep(2)
```

Comment:
Generally it would be nice to avoid arbitrary 'sleep's in the code, but I appreciate that we're doing weird network magic here with an uncooperative third-party system, so for the sake of having things work I'm happy to leave this in.